aggregator_core/src/datastore/models.rs

```diff
@@ -451,6 +451,8 @@ pub enum AggregationJobState {
     Finished,
     #[postgres(name = "ABANDONED")]
     Abandoned,
+    #[postgres(name = "DELETED")]
```

Comment:
This matches the [`COLLECTION_JOB_STATE` enum](https://github.com/divviup/janus/blob/2b64b2a13fd1fa3e4173ac56cab4dd0a900218ed/db/00000000000001_initial_schema.up.sql#L305-L311). My thinking is that if and when we take job deletion into account in garbage collection, we might want to distinguish between abandoned and deleted. For instance, maybe we want to aggressively clean up deleted jobs, but want to keep abandoned ones around for investigation, since abandonment indicates an unusual condition whereas deletion is more routine.